docs/03-pages/02-api-reference/02-functions/use-amp.mdx

```diff
@@ -16,7 +16,7 @@ description: Enable AMP in a page, and control the way Next.js adds AMP to the p
 
 To enable AMP, add the following config to your page:
 
-```jsx
```

Comment:
This is on a page so it should probably be jsx. Also maybe give it a name

Reply:
Fixed!